app/src/main/java/no/usn/mob3000/domain/usecase/social/userProfile/FetchUserByIdUseCase.kt

```diff
@@ -0,0 +1,17 @@
+package no.usn.mob3000.domain.usecase.social.userProfile
+
+import no.usn.mob3000.data.repository.social.UserRepository
+import no.usn.mob3000.domain.model.auth.UserProfile
+/**
+ * This use case retrieves the user's profile information.
+ * @param userRepository The repository for user operations.

```

Comment:
There should be an empty KDoc line between KDoc properties and descriptive content.